WordPress/Sniffs/PHP/FunctionEnqueuedSniff.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPress\Sniffs\PHP;
+
+use WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * Based off the StrictInArraySniff.php Sniff, THis checks the Enqueued 4th Parameter to make sure a Version is available.
+ * The Enqueued functions are:
+ * wp_register_script
+ * wp_enqueue_script
+ * wp_register_style
+ * wp_enqueue_style
+ * wp_register_style
+ *
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1146
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.14.0 This new sniff will check for a version in an Enqueued WP function
```

Comment:
All of the `@since` version numbers should probably be updated to 0.15.0 (presumably the next version that will be released).